config.omero

```diff
@@ -0,0 +1,16 @@
+# By default, we enable 'minimal_webapp' only
+config append omero.web.apps '"minimal_webapp"'
+
+# Add a link to the app from webclient toolbar
+config append omero.web.ui.top_links '["Minimal-Webapp", "minimal_webapp_index", {"title": "Open app in new tab", "target": "_blank"}]'
+
+# Uncomment this out to enable, if app has been installed
+# config append omero.web.apps '"react_webapp"'
+
+config set omero.web.debug True
+
+# enable CORS so e.g. react-webapp can connect to OMERO.web on different origin
+config append omero.web.apps '"corsheaders"'
```

Comment:
this assumes that django-cors-headers has been installed.
maybe add a link to https://docs.openmicroscopy.org/omero/5.6.0-m4/sysadmins/unix/install-web.html